herbert_core/classes/instrument_classes/@Goniometer/Goniometer.m

```diff
@@ -188,7 +188,7 @@
     %======================================================================
     % partial load and angular transformations.
     methods
-        % ANGULAR TRANSFORMATIONS (is it necessary? A bit overcomplecated
+        % ANGULAR TRANSFORMATIONS (is it necessary? A bit overcomplicated
```

Comment:
be nice if the reader understood what was meant here. Is this a note to the developer in future, or a warning to the user, or what. A basic description of what is over-complicated is needed.